index.d.ts

```diff
@@ -231,6 +232,7 @@ export interface Rule {
  * The Style is the main interface and the root for all other interfaces.
  */
 export interface Style {
+  name: string;
```

Comment:
or make it optional by writing

```
  name?: string;
```
?

Reply:
Thought about that. But as its required for a rule i decided to make it required for the style to...